components/CancellationList.vue

```diff
@@ -181,8 +181,14 @@ export default {
         text: `${vue.$t("cancelList.header.prefecture")}:`,
         value: "prefecture",
       },
-      { text: `${vue.$t("cancelList.header.city")}:`, value: "city" },
-      { text: `${vue.$t("cancelList.header.ward")}:`, value: "ward" },
+      {
```

Comment:
Can we undo accidental layout changes? Make sure you have both prettier and eslint configured.

Reply:
@ann-kilzer reverted! 😺